wdio.conf.js

```diff
@@ -339,6 +326,16 @@ exports.config = {
 };
 
 if (process.env.SAUCE) {
+  const sauceCapabilities = Object.assign({}, chromeCapabilities, {
```

Comment:
We need to handle firefox capabilities as well, correct?

Reply:
good catch